Gemfile

```diff
@@ -0,0 +1,31 @@
+source 'https://rubygems.org'
+gem 'bcrypt', '~> 3.1.7'
+gem 'bootsnap', require: false
+gem 'carrierwave'
+gem 'devise'
+gem 'grape'
```

Comment:
Let's remove these gems from this PR, and add them in the required PR where we implement these.

Reply:
Fixed it Apu. Thank you.